portfolio/src/main/webapp/index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8">
     <title>Nick's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDuc9LUfW2xFbz9gmQMlQzQbLs-3_f98WA"></script>
     <script src="script.js"></script>
   </head>
-  <body id = "background" onload = "getGame()">
+  <body id = "background" onload = "createMap();">
```

Comment:
Regarding space-padding your `=` signs, you should choose a consistent style and stick with it, rather than mixing the two.  I'd recommend no spaces -- until today I didn't think that padding the `=` in HTML was even allowed.